src/views/ont/OntReception.vue

```diff
@@ -36,13 +39,18 @@ export default {
   data () {
     return {
       barcodes: [],
-      plates: {}
+      plates: {},
```

Comment:
Think plates might not need to be a data attribute if we use the plates returned from `getSequencescapePlates`